egs2/how2/sum1/run_sum.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
We basically put one `run.sh` in the directory and put the others in `local`.
So, please rename it as run.sh and move `run_asr.sh` to `local/run_asr.sh`